src/components/ErrorMessage.tsx

```diff
@@ -15,25 +16,71 @@ const Container = styled.View`
   display: flex;
   align-items: center;
 `
-
-const ErrorText = styled.Text`
+const ErrorTitle = styled.Text`
+  padding: ${props => `${props.theme.spacings.md} ${props.theme.spacings.xl}`} 0 ${props => props.theme.spacings.xl}};
+  font-size: ${props => props.theme.fonts.headingFontSize};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  color: ${props => props.theme.colors.primary};
+  text-align: center;
+`
+const ErrorText = styled.Text<{ centered?: boolean }>`
   padding: ${props => `${props.theme.spacings.md} ${props.theme.spacings.xl}`};
   font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  text-align: ${props => (props.centered ? 'center' : 'left')};
+`
+const NetworkErrorWrapper = styled.View`
+  background-color: ${prop => prop.theme.colors.background};
+  height: 100%;
+  align-items: center;
+`
+const IconStyle = styled.View`
+  width: 33%;
+  height: auto;
+  align-items: center;
+`
+const UpperSection = styled.View`
+  width: 140%;
+  height: 67%;
+  background-color: ${props => props.theme.colors.networkErrorBackground};
+  border-bottom-left-radius: ${hp('60%')}px;
+  border-bottom-right-radius: ${hp('60%')}px;
+  margin-bottom: ${props => props.theme.spacings.xxl};
+  justify-content: center;
+  align-items: center;
+  text-align: center;
 `
 
 interface ErrorMessageProps {
   error: Error | null
   refresh: () => void
+  contained?: boolean
 }
 
-const ErrorMessage = ({ error, refresh }: ErrorMessageProps): JSX.Element | null =>
-  error && (
+const ErrorMessage = ({ error, refresh, contained }: ErrorMessageProps): JSX.Element | null =>
+  error &&
```

Comment:
Optional/personal preference: Imo it is much easier to read if you use brackets and if statements.

Reply:
I'm not 100% sure what you mean - the conditional render was already "there", I just amended the second part. Do you want me differentiate between the cases outside the return statement?

Reply:
If I were to write this component from scratch I'd do something like
``` tsx
 const ErrorMessage = ({ error, refresh, contained }: ErrorMessageProps): JSX.Element | null => {
  if (!error) {
    return null
  }
  if (!contained && error.message === NetworkError) {
    return xyz
  }

  return asdf
}
``
But as I said, this is just personal preference because I find it easier to see the used conditions. So your choice if you just want to keep the current or change it.

Reply:
I'll hold off on these changes until it's clear whether all errors get the new layout, then implement them.

Reply:
All error can get the new layout.

Reply:
Since there are no icons for the other errors, should the icon just be removed for other errors? Granted, the highlighted area does look a bit empty then.

Reply:
Will be done in a separate issue :)

Reply:
was the latest commit not needed then?

Reply:
Ah I see, you already did it. Yeah then this is fine, thanks! So no additional issue is needed :)